src/index.ts

```diff
@@ -76,29 +76,33 @@ class VueBreadcrumbs implements PluginObject<ComponentOptions<Vue>> {
             },
             this.$breadcrumbs.map((crumb: RouteRecord, index: number) => {
               if (crumb?.meta?.breadcrumb) {
-                return createElement(
-                  'li',
-                  {
-                    class: {
-                      'breadcrumb-item': true
-                    },
-                    props: {
-                      key: index
-                    }
-                  },
-                  [
-                    createElement(
-                      'router-link',
-                      {
-                        props: {
-                          to: { path: this.getPath(crumb) },
-                          tag: index !== this.$breadcrumbs.length - 1 ? 'a' : 'span'
-                        }
+                let label = this.getBreadcrumb(crumb.meta.breadcrumb);
```

Comment:
Could you change a little here

```diff
+          if (crumb?.meta?.breadcrumb && this.getBreadcrumb(crumb.meta.breadcrumb)) {
-              let label = this.getBreadcrumb(crumb.meta.breadcrumb);
-               if (label !== '') {
```

Reply:
My thought was to minimise the repetitive call to this.getBreadcrumb. By assigning it to a variable, it's then reused further down

Reply:
That's right, I just ask you to write down the condition otherwise as indicated above, the meaning and logic remain the same theme